.github/workflows/test-go.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
 
     - name: Run Go Tests
       run: |
-        MYSQL_TEST=1 make test-go
+        NETWORK_TEST=1 REDIS_TEST=1 MYSQL_TEST=1 make test-go
```

Comment:
Not a huge fan of these, but I get it.

We did remove `REDIS_TEST`, so we should remove it from here. I would also like to remove `MYSQL_TEST` entirely and just keep working on making things better.

Reply:
Yeah, I agree with you, it's not very discoverable, but as mentioned in the description the approach I would've preferred was quite an undertaking, especially without fully understanding that part of the code. I believe the `REDIS_TEST` env var is still used in the `server/sso` tests.

Reply:
Oh... darn, missed that :( Allllrighty then